Automattic/vip-go-mu-plugins
# Search: Do not offload to ES if improper orderby is passed in

## Description
When an improper `orderby` is passed into an ES query, it causes the ES query to fail with `No mapping found for <invalid orderby value> in order to sort on`. This prevents the query from being offloaded to ES in the first place since it fails anyways.

## Changelog Description

### Changed
- Search: Invalid orderby parameters will not offload to ES

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

On a sandbox, put an invalid orderby ES query to be offloaded:
```
add_action( 'init', function() {
	$args = array(
		'post__in' => [ 1, 2 ],
		'posts_per_page' => 10,
		'post_status' => 'publish',
		'ignore_sticky_posts' => true,
		'no_found_rows' => true,
		'orderby' => 'post__in',
		'ep_integrate' => true,
	);
	$recent_posts = new WP_Query( $args );
} );
```
In Search Dev Tools, you'll see the error mentioned in the description in the response. Once the PR is applied, the query will no longer be offloaded to ES.
